docs/source/using-pbft-consensus.rst

```diff
@@ -21,6 +21,11 @@ restricted membership. It has the following requirements:
   using the on-chain settings ``sawtooth.consensus.algorithm.name`` and
   ``sawtooth.consensus.algorithm.version``.
 
+  - For the name and version number, see ``Cargo.toml`` in
```

Comment:
The name and version to use are not in the Cargo.toml file; they're specified in the `name` and `version` methods of the `src/engine.rs` file. Those strings are what the settings must match.

Reply:
Hmmm.  I was trying to match the information in the create-network procedures in the Sawtooth App Dev and Sys Admin guides. (See the tip in step 5 of "[Creating the Genesis Block](https://sawtooth.hyperledger.org/docs/core/nightly/master/sysadmin_guide/creating_genesis_block.html#)".)

I could change the info here, but would be less happy about changing the Sawtooth core procedures). I'm loath to ask an administrator to dig around in Rust source code for the PBFT version number.

Reply:
Sorry, I don't think my previous comment was very helpful. The version is fine (using the first two digits from the Cargo.toml), since that should be updated in the code to match whatever's in the Cargo.toml. The `name` value (`pbft`), though, isn't anywhere in the `Cargo.toml`. So it would be sufficient to just say, "It must be `name=pbft` for PBFT."

Reply:
That sounds good. Thanks!

Reply:
Updated.